### ISSUE TITLE
The sim loss of domain adaptation

### ISSUE THREAD

**Issue by Lucksong** [tensorflow/models#6625]
Hi @bousmalis ,thanks for your codes about domain adaption at models/research/domain_adaption/domain_separation, it's very cool. But why do you add limits to it's mmd loss : loss_value = tf.maximum(1e-4, loss_value) and cost = tf.where(cost>0,cost,0)?
You mean the loss must be posative and must backward with at least 1e-4? And can provid the hyperparameters
Thanks !

**Comment by Lucksong** [comment]
Are these two codes not contradictory? Can provide the weights of mmd loss?
Thanks

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
